### PR TITLE
Update AWS OIDC examples

### PR DESCRIPTION
**Pull request by pierskarsenbarg**
Now using the tls provider to get the thumbprints required for the oidcprovider

### CODE REVIEW

File: aws-py-oidc-provider-pulumi-cloud/__main__.py [hunk 1]
```diff
@@ -10,41 +11,15 @@
...
-x509 = OpenSSL.crypto.load_certificate(OpenSSL.crypto.FILETYPE_PEM, last_certificate)
-thumbprint = (x509.digest('sha1').decode()).replace(":", "")
+certificate = tls.get_certificate(url=oidc_idp_url)
 
 # Create an OIDC identity provider
 print("Creating OIDC provider...")
 oidc_provider = iam.OpenIdConnectProvider("oidcProvider",
     client_id_lists=[audience],
-    thumbprint_lists=[thumbprint],
+    thumbprint_lists=[certificate.certificates[0].sha1_fingerprint],
```
Comment (desteves) [inline aws-py-oidc-provider-pulumi-cloud/__main__.py:22]: Root CA only

Comment (pierskarsenbarg) [reply aws-py-oidc-provider-pulumi-cloud/__main__.py:22]: If you run this:

```typescript
import * as tls from "@pulumi/tls";

export = async () => {
    const certs = tls.getCertificateOutput({
        url: "https://api.pulumi.com/oidc"
    });

    return {
        thumbprint: certs.certificates[0].sha1Fingerprint
    }
}
```

the thumbprint is the same as what's in IAM > Identity Providers > api.pulumi.com/oidc 

and look at the thumbprint value.

Comment (desteves) [reply aws-py-oidc-provider-pulumi-cloud/__main__.py:22]: `9e99a48a9960b14926bb7f3b02e22da2b0ab7280`

**Review by desteves — APPROVED** [review]
LGTM

**Pull request merged**